src/test/java/com/ounitech/wemove/repositories/MemberRepositoryTest.java

```diff
@@ -5,13 +5,15 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.TestPropertySource;
 
 import java.util.Optional;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 
 @DataJpaTest
+@TestPropertySource(locations = "classpath:application-test.properties")
```

Comment:
Test is still failing, you forgot to commit application-test.properties :wink:

Reply:
Fixed it